particle-iot/device-os
# Enables UPSV=1 low power mode for R510 when idle for >=9.2s

## ⚠️ NOTE: First MERGE referenced MUXER PR, then before merging this PR, REBASE WITH MUXER master checked out and pulled.

### Problem

- R510 modem consumes more power than R410 modem by design.  This is unexpected behavior for some customers migrating from R410 to R510.

### Solution

- Reduce power consumption for R510 based devices by enabling UPSV=1 mode which will automatically reduce power consumption after 9.2s of being idle.  Note: Some extreme cases of the modem requiring over 2 minutes to drop into a low power state have been observed and are being investigated.  Typically though, if the AT interface is idle, the modem will automatically reduce power consumption.

### Steps to Test

- Compile `APP=tinker-serial1-debugging` and observe after device is idle, power consumption drops from 30-38mA avg. down to 8-12mA avg.
- Integration test - `device-os-test run bsom wiring/no_fixture_long_running -v -- -fixture`
- Integration test - `device-os-test run boron wiring/no_fixture_long_running -v -- -fixture`
- Integration test - `device-os-test run esomx wiring/no_fixture_long_running -v -- -fixture`

### References

- https://github.com/particle-iot/gsm0710muxer/pull/11

---

### Completeness
